src/cli/cmd/list.rs

```diff
@@ -32,7 +61,14 @@ impl Flake {
     }
 
     fn url(&self) -> String {
-        format!("{}/flake/{}/{}", FLAKEHUB_WEB_ROOT, self.org, self.project)
+        let mut url = Url::parse(FLAKEHUB_WEB_ROOT).unwrap();
```

Comment:
Would be swell if this could be an `expect("web root should have been a valid url")` instead of an `unwrap`, but I won't block on it.